binie.md

```diff
@@ -0,0 +1,84 @@
+This approach ensures that a single document file can be used across different platforms and applications, maintaining its layout integrity and visual fidelity.
+
+## Specification
+
+The Binie file consists of a document header followed by a series of elements. Each element can be either text or an image, each with its specific header and data.
+
+Every integer in the Binie specification is encoded in Little Endian format.
+
+### Binie Header
+
+Binie documents begin with a standard header that includes essential metadata about the document:
+
+| Offset (bytes) | Length (bytes) | Content                 |
+|----------------|----------------|-------------------------|
+| 0              | 256            | Document Title          |
+| 256            | 4              | Document width          |
+| 260            | 4              | Document height         |
+| 264            | 4              | Element count           |
+
+### Element Header
+
+Following the header, Binie documents contain a series of elements, each with a defined structure.
+Each element in the document (text or image) starts with a standard header.
+
+| Offset (from element start) | Length (bytes) | Content            |
+|-----------------------------|----------------|--------------------|
+| 0                           | 4              | Element type       |
+| 4                           | 4              | Element data length|
+
+### Text Element
+
+Text elements have their specific header followed by the actual text. 
+The element type for text elements is identified by the value `1`.
+
+#### Text Element Header
+
+| Offset (from text element start) | Length (bytes) | Content             |
+|----------------------------------|----------------|---------------------|
+| 0                                | 8              | Common header       |
+| 8                                | 8              | Position            |
+| 16                               | 256            | Font                |
+| ...                              | ...            | Additional settings |
+
+#### Text Data
+
+Follows immediately after the text element header.
+
+### Image Element
+
+Image elements contain image data.
+The element type for image elements is identified by the value `2`.
+
+#### Image Element Header
+
+| Offset (from image element start) | Length (bytes) | Content            |
+|-----------------------------------|----------------|--------------------|
+| 0                                 | 8              | Common header      |
+| 8                                 | 8              | Position           |
+| ...                               | ...            | Additional settings|
+
+#### Image Data
+
+Follows immediately after the image element header.
+
+### Text Alignment
```

Comment:
I would probably put the values range in the table row associated with the alignment (more contextual). But I'll still merge this one.

Reply:
https://github.com/hivesolutions/colony-print/pull/6